src/module/actor/character/crafting/formula.ts

```diff
@@ -78,10 +78,15 @@ export class CraftingFormula implements CraftingFormulaData {
     get description(): string {
         return this.item.description;
     }
+
+    get quickAlchemy(): boolean {
```

Comment:
I don't think this is the right place for it. There are ways of being able to quick alchemy certain formulas that are actually not alchemical. This is probably something that should be driven by the crafting entries instead.

https://2e.aonprd.com/Feats.aspx?ID=123

This feat makes the potions into elixers, but I suspect the transformation of potion -> elixer would be done as part of the quick crafting pipeline itself. Unless you got a different idea, such as storing data at the formula level on whether or not its an improbably elixer or not?

Reply:
Of course there has to be an exception... and they don't even get the alchemical trait. 
Problem is that there is currently no way to alter a formula permanently as they are strict references to a compendium item.
Maybe with a choice set on the feat to select potions that sets a flag on the actor which could be utilized to collect all quick alchemy items in `getData`.

I think I'll revert to the toggle while I think about it.

Reply:
Being an elixir forces the alchemical trait, as per the elixir trait I think.

I don't think you have to change what you filter at least for this draft, probably only where you filter it. Perhaps a crafting entry can have a canQuickAlchemy(item) and then you allow it if it at least one matches. Or maybe the getter is fine.

Checking via the crafting entry would also handle the cases where you have advanced alchemy but not quick alchemy (herbalist).

Reply:
Crafting entries currently only include "prepared" formulas for daily crafting, but quick alchemy can be used with any known formula, so putting the check on the entry doesn't really make sense.
The`hasQuickAlchemy` check happens in the template so the button only shows up when the character has the action.

But leaving this change in will make it impossible to craft items from the feat you postetd because the formula traits can't be altered. One solution would be to have a `Known Formulas` list per crafting entry where formula traits could be added based on character feats.

Reply:
You mighta misunderstood. I don't mean via the formula in crafting entry, I meant maybe a crafting entry could have a method that receives a formula that then lets you know if it can do quick alchemy with that formula (regardless of whether its been added there or not).

Crafting Entries (terrible name) are basically advanced alchemy features. You use quick alchemy with an advanced alchemy feature. I think it makes sense for them to decide if a formula can be quick alchemy'd. In fact, you can't quick alchemy at all if there are zero crafting entries.

Reply:
I see. I didn't realize crafting entries are just used for advanced alchemy. That still leaves the question how a potion formula could be flagged as eligible for quick alchemy if it's base on a compendium reference.
For now I've reverted back to the global toggle to not make those edge cases impossible.